packages/ckeditor5-watchdog/src/editorwatchdog.ts

```diff
@@ -323,6 +352,110 @@ export default class EditorWatchdog<TEditor extends Editor = Editor> extends Wat
 	}
 }
 
+class EditorWatchdogInitPlugin {
```

Comment:
API docs or inline comments at least for this class, even though it is private internal thingy.